README.md

```diff
@@ -0,0 +1,76 @@
+# Bazel Rules Authors SIG
+
+Bazel is a full-stack monorepo build & test tool from Google. <https://bazel.build>
```

Comment:
```suggestion
Bazel is a full-stack build & test tool from Google. <https://bazel.build>
```

Just thinking that while this is preferred, it's not required, so I think we can remove it to not risk alienating folks